.github/workflows/monorepo-docker.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           submodules: recursive
 
       - name: remove submodule locks
-        run: git submodule foreach rm yarn.lock
+        run: git submodule foreach 'test -f yarn.lock && rm yarn.lock || echo "No yarn.lock file in this submodule"'
```

Comment:
Seems like a fine change to me, just curious were you seeing errors without the test?

Reply:
yup, yarn.lock not found in solidity/lib/forge-std